hostdb/hostdb.go

```diff
@@ -76,3 +77,19 @@ func (h *Host) NetAddress() string {
 	}
 	return h.Announcements[len(h.Announcements)-1].NetAddress
 }
+
+// NetAddress returns the host's last announced NetAddress, if available.
+func (h *Host) LastKnownSettings() (rhpv2.HostSettings, time.Time, bool) {
```

Comment:
wrong docstring. Also, weren't we going to move this out of `hostdb`?

Reply:
this was mistakenly reintroduced because of a merge I assume, removed it